bitmex-stream/src/lib.rs

```diff
@@ -63,3 +65,18 @@ pub fn subscribe<const N: usize>(
 
     stream.boxed()
 }
+
+#[derive(Debug, Copy, Clone)]
+pub enum Network {
+    Mainnet,
+    Testnet,
+}
+
+impl Network {
+    pub fn to_url(&self) -> String {
+        match self {
+            Network::Mainnet => "www.bitmex.com".to_string(),
+            Network::Testnet => "testnet.bitmex.com".to_string(),
```

Comment:
nit: `www.` prefix only in one type of network? it seems like it's probably not needed in mainnet either?

Reply:
I tried without and it is. I don't have an easy answer why.

Reply:
The subdomain of `www` is  not voluntarily, it's the official url they are running their API on :D 

See https://testnet.bitmex.com/api/explorer/ vs https://www.bitmex.com/api/explorer/